app/src/pages/metamine/PixelUnit/PixelUnit.vue

```diff
@@ -0,0 +1,18 @@
+<template src="./PixelUnit.html"></template>
+<script src="./PixelUnit.js"></script>
+<style scoped>
```

Comment:
Similar to discussion in PR #142 about scoped styling
Possibly there could be a single _tools.scss file in assets/css/modules that handles the class stylings of all of the tools so we don't have to make individual files for each? Again, not sure of the best way to handle this @tholulomo

Reply:
I agree, I'll suggest moving all style assets into the assets/css folder and creating a _tools.scss file.